inst/components/scss/sidebar.scss

```diff
@@ -318,6 +325,19 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sideba
         left: unset;
       }
 
+      .sidebar {
+        opacity: var(--_sidebar-mobile-opacity);
+        max-width: var(--_sidebar-mobile-max-width);
+        box-shadow: var(--_sidebar-mobile-box-shadow);
+      }
```

Comment:
If these are inherited properties, do we need to default the CSS variable to `inherit`?

Reply:
Good news: none of these are inherited properties. I'll added defaults, but I think we're okay here.